Gems/ScriptCanvas/Code/Include/ScriptCanvas/Debugger/Debugger.cpp

```diff
@@ -68,7 +73,7 @@ namespace ScriptCanvas
                 }
 
                 // \todo send a connection denied message
-                // AzFramework::TargetManager::Bus::Broadcast(&AzFramework::TargetManager::SendTmMessage, sender, Message::Denied());
+                // remoteToolsInterface->SendRemoteToolsMessage(sender, Message::Denied());
```

Comment:
remove commented out code

Reply:
oh, I see, this is script canvas. maybe they left in here for an example to whoever finally implements. Maybe put "Example: " before the code, otherwise it looks like commented out code that should be deleted.

Reply:
This is just what ScriptCanvas had beforehand. Any conversion work done here is best effort. Leaving this as is.